src/Pos/Context/Functions.hs

```diff
@@ -4,6 +4,7 @@ module Pos.Context.Functions
        (
          -- * Genesis
          GenesisUtxo(..)
+       , mkGenesisTxpContext
```

Comment:
Looks weird indeed. `Context` reexports `GenesisUtxo` because it was the part of the context. And it is. I guess.